src/context/Wishlist-Context.jsx

```diff
@@ -0,0 +1,49 @@
+import { createContext, useContext, useState } from "react";
+import axios from "axios"
+
+
+const addToCartClickHandler = async (product, setItemsInCart, encodedToken, navigate) => {
+   
+    if(encodedToken){
+      try {
+        const cartItems = await axios.post( `/api/user/cart`, {product} ,
+          { headers: { authorization: encodedToken } }
+        );
+        setItemsInCart(cartItems.data.cart)
+      } catch (err) {
+        alert("error from server", err)
```

Comment:
For error use toasts, it was suggested by a MA in a code review session

Reply:
yeah will add after finishing  it.